### DISCUSSION TITLE
Repository Structure

### DISCUSSION THREAD

**Discussion by psteinroe** [supabase/supabase#9234] [Feature Requests]
Hey,

while its great how fast supabase ships new features, it can now be a struggle to find the right place to ask a question or report a bug. Most prominent example are the js libraries, where I have seen hundreds of issues being transferred between supabase-js and e.g. the postgrest-js repository. Also, I am never sure what to put into a discussion versus an issue on the main repo, maybe it would make sense to move e.g. the dashboard into an own repo?

**Comment by thorwebdev** [comment]
The idea of the main repo is that you don't have to think where to go, but just open an issue or question on the main repo.

The idea is:
- Go to https://github.com/supabase/supabase/issues
- Click "New Issue"
- Choose the right category
- And that's it.
- The team will then take care of moving it to a different repository if necessary.

We have a section in the README explaining the different channels also: https://github.com/supabase/supabase#community--support

Have you seen this before? If so, do you think it's not clear enough?
